src/Agent/NewRelic/Agent/Core/Transactions/Transaction.cs

```diff
@@ -775,6 +775,14 @@ public void RollupTransactionNameByStatusCodeIfNeeded()
             }
         }
 
+        public void SetRequestMethod(string requestMethod)
+        {
+            if (requestMethod == null)
+                throw new ArgumentNullException(nameof(requestMethod));
```

Comment:
Not sure what are coding standards say, but I would add braces to all logic flow control

Reply:
Our coding standard do prefer braces over the shorthand.